### ISSUE TITLE
TicketOverview: DynamicField with Link and Value Replacement did not add Value

### ISSUE THREAD

**Issue by hcamminadi** [RotherOSS/otobo#805]
**Situation:**
You have a dynamic field for "Ticket" with option "Show link". The field is named "myFieldName" and labeled "my field name". 
"Show Link" contains the following value:
https://<anyurl>/view.php?id=[% Data.DynamicField_MyFieldName %]
The value in a sample ticket is '5555'

AgentTicketZoom Ticket-Information shows the Link and generates the following well working html:

```html
<label>my field name</label>
      <p class="Value">
          <span title="5555"><a href="https://<anyurl>/view.php?id=5555" target="_blank" class="DynamicFieldLink">5555</a></span>
      </p>
```
The problem is TicketOverview as Small.pm, Medium.pm and Preview.pm did not enhance the "DynamicField_" to the variable name. It seems that this is an issue as TicketInformation.pm contains the following lines:

```perl
    # Include unique parameter with dynamic field name in case of collision with others.
    #   Please see bug#13362 for more information.
    "DynamicField_$Field->{Name}" => $LinkValue, 
```
Small, Medium and Preview.pm did not contain the same prefix. 
Adding the prefix e.g. in Small.pm line 1700 would solve the problem:

```perl
                if ( $ValueStrg->{Link} ) {
                    $LayoutObject->Block(
                        Name => 'RecordDynamicFieldLink',
                        Data => {
                            Value                       => $ValueStrg->{Value},
                            Title                       => $ValueStrg->{Title},
                            Link                        => $ValueStrg->{Link},
                            "DynamicField_$DynamicFieldConfig->{Name}" => $ValueStrg->{Title},
                        },
                    );
                }
```

**Comment by svenoe** [comment]
Hi again and thanks again. ;)

We will also have a look at this.

**Comment by bschmalhofer** [comment]
Reopened because the issue is still at 'Needs triage'

**Comment by stefanhaerter** [comment]
Closed with merging #3514
